lib/rules/no-underscore-dangle.js

```diff
@@ -126,16 +131,40 @@ module.exports = {
         }
 
         /**
-         * Check if function has a underscore at the end
+         * Check if function parameter has a dangling underscore.
+         * @param {ASTNode} node node to evaluate
+         * @returns {boolean} true if function parameter name has a dangling underscore.
+         * @private
+         */
+        function checkForDanglingUnderscoreInFunctionParameter(node) {
+            if (!allowFunctionParams) {
+                node.params.forEach(param => {
+                    const identifier = param.name;
```

Comment:
It would be nice to explicitly check whether the node is `Identifier` type.

Reply:
> It would be nice to explicitly check whether the node is `Identifier` type.

@mdjermanovic 
First of all, thank you for leaving a comment. 😄 
This part is a little confusing. Is it correct to check if the type of `param` is Identifier? Or is it asking to check the type of Node itself?
> Like this,
```js
function checkForDanglingUnderscoreInFunctionParameters(node) {
            if (!allowFunctionParams) {
                node.params.forEach(param => {
                    if (param.type === Identifier) {  // Is it Right???
                        const identifier = param.name;
                        ...
                    }
```

Reply:
Sorry for my confusing post!

> Is it correct to check if the type of param is Identifier?

Yes, exactly that: `param.type === "Identifier"`

Also, with that condition, it wouldn't be necessary to additionally check whether `param.name` is `undefined` or not.